### PR TITLE
Adds --stdin-filename back to changelog

### PR DESCRIPTION
**Pull request by austinpray-mixpanel**
Looks like this went missing https://github.com/psf/black/commit/dea81b7ad5cfa04c3572771c34af823449d0a8f3#diff-729efdd61772b108539268bdbfd7472521bdc05a7cae6113f62ed2649a3ad9c7

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -30,6 +30,9 @@
 
 - `--diff` correctly indicates when a file doesn't end in a newline (#1662)
 
+- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules.
+  Works very alike to flake8's `--stdin-display-name` (#1780)
```
Comment (JelleZijlstra) [inline CHANGES.md:34]:
```suggestion
- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules (#1780)
```
The sentence about flake8 is more suited for documentation, not a changelog.

Comment (austinpray-mixpanel) [reply CHANGES.md]: Sure, I just copied was was in there previously

File: CHANGES.md [hunk 2]
```diff
@@ -30,6 +30,8 @@
 
 - `--diff` correctly indicates when a file doesn't end in a newline (#1662)
 
+- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules (#1780)
```
Comment (JelleZijlstra) [inline CHANGES.md:33]:
```suggestion
- Added `--stdin-filename` argument to allow stdin to respect `--force-exclude` rules
  (#1780)
```
Fix lint. Sorry for the churn, but while we're at it we might as well improve the wording.

**Comment by austinpray-mixpanel** [comment]
Thank you! 👍

**Comment by ichard26** [comment]
Well thank you @austinpray-mixpanel for noticing and fixing the issue!

**Pull request merged**